.github/workflows/build-embeddable-progressive-loan-jar.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       - name: Setup Gradle
         uses: gradle/actions/setup-gradle@94baf225fe0a508e581a564467443d0e2379123b # v4.3.0
       - name: Build Embeddable Jar
-        run: ./gradlew --no-daemon --console=plain :fineract-progressive-loan-embeddable-schedule-generator:shadowJar
+        run: ./gradlew --no-daemon --console=plain :fineract-progressive-loan-embeddable-schedule-generator:shadowJar && ./gradlew --no-daemon --console=plain :fineract-progressive-loan-embeddable-schedule-generator:test
```

Comment:
You can move the execution of the tests into its own named steps, like: "Pick up the JAR filename" or " Build Sample Application"